astropy/modeling/fitting.py

```diff
@@ -2103,8 +2103,7 @@ def model_to_fit_params(model):
 
 
 @deprecated(
-    since="5.1",
-    message="private method: _model_to_fit_params has been made public now"
+    since="5.1", message="private method: _model_to_fit_params has been made public now"
```

Comment:
I just don't get it, but ok.

Reply:
the main thing is consistency by black, right :trollface: